### PR TITLE
Fixes issue #306 

### PR DESCRIPTION
**Pull request by KozmikNano**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -93,7 +93,10 @@ const askForSecrets = async () => {
 	if (config.get("discordBot")) {
 		dc = new discord.Client();
 		dc.login(discordBotToken).catch(()=>{
-			console.warn("There was an error when trying to log in using the provided Discord bot token."); //handle empty tokens gracefully
+			if (config.BotToken != ""){
+			console.warn("There was an error when trying to log in using the provided Discord bot token. If you didn't enter a token this message will go away the next time you run this program!"); //handle empty tokens gracefully
+				
+			};
```
Comment (rob9315) [inline main.js:99]:
```suggestion
			console.warn("There was an error when trying to log in using the provided Discord bot token. If you didn't enter a token this message will go away the next time you run this program!"); //handle wrong tokens gracefully
```

File: main.js [hunk 2]
```diff
@@ -93,7 +93,10 @@ const askForSecrets = async () => {
 	if (config.get("discordBot")) {
```
Comment (rob9315) [inline main.js:93]:
```suggestion
        config.discordBot = config.BotToken !== "" ? false : config.get("discordBot");
	if (config.get("discordBot")) {
```

**Review by rob9315 — CHANGES_REQUESTED** [review]
Please don't create a Bot Object at all if the token is empty, saves resources.

File: main.js (hunk 2 above)
Comment (KozmikNano) [reply main.js:93]: is that all that needs to be added, or is there stuff that has to be re-worded/added?

Comment (rob9315) [reply main.js:93]: That should do it

Comment (KozmikNano) [reply main.js:93]: ok thank you, I will add that line then test

File: main.js [hunk 3]
```diff
@@ -90,6 +90,7 @@ const askForSecrets = async () => {
 		console.clear();
 	}
 
+        config.discordBot = config.BotToken !== "" ? false : config.get("discordBot");
```
Comment (rob9315) [inline main.js:93]:
```suggestion
        config.discordBot = discordBotToken !== "" ? false : config.get("discordBot");
```

**Review by rob9315 — CHANGES_REQUESTED** [review]

**Comment by rob9315** [comment]
fixes #306

**Pull request merged**